### ISSUE TITLE
Accessibility Bug: When the page number is selected, the selected information ‘page 4 is selected’ not conveyed(screen reader)

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3129]
Fundamental core component: "Pagination"

Description: When the page number is selected, the selected information ‘page 4 is selected’ not conveyed(screen reader)

Expected: The state ‘selected’ information should be conveyed(screen reader) when a page number is selected.

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/91557950-0560aa80-e953-11ea-83a3-ffe13753437e.png)

Note: - Issue persists for all the page numbers present within the ‘pagination’ control.

**Comment by Sanjay-1312** [comment]
![Pag num](https://user-images.githubusercontent.com/78714518/119773735-b3978180-bede-11eb-9eb6-157d0d2ed0e7.png)



Fundamental core component: "Pagination"

Description: When the page number is selected, the selected information ‘page 4 is selected’ not conveyed(screen reader)

Expected: The state ‘selected’ information should be conveyed(screen reader) when a page number is selected.



Hello Colleagues,

The issue still persists.
Please fix the issue
JAWS 2021

Thanks and Regards
Sanjay

**Comment by manu-kr** [comment]
@Sanjay-1312 The URL you're verifying seems wrong, Please use the below URL for testing this defect.

https://deploy-preview-5468--fundamental-ngx.netlify.app/#/core/pagination

Issue is we shouldn't use aria-selected with 'a' link tag, so currently we are using 'aria-current' instead of 'aria-select' and it will read selected page current page.

For more information please refer below links.
https://www.a11ymatters.com/pattern/pagination/
https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-pagination

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is fixed hence closing the ticket.
URL: https://deploy-preview-5468--fundamental-ngx.netlify.app/#/core/pagination
JAWS 2021

Thanks and Regards
Sanjay
